src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
```

Comment:
please use the full name instead of an abbrevation - it hurts readability. I had the impression that a custom base class was created called `Dmf` until I checked the imports.